fe/fe-core/src/main/java/com/starrocks/catalog/TabletInvertedIndex.java

```diff
@@ -400,7 +400,7 @@ public void addTablet(long tabletId, TabletMeta tabletMeta) {
         try {
             tabletMetaMap.putIfAbsent(tabletId, tabletMeta);
 
-            LOG.debug("add tablet: {}", tabletId);
+            LOG.info("add tablet: {}, meta: {}", tabletId, tabletMeta);
```

Comment:
I'm afraid this will generate a lot of log when creating tables.

Reply:
Yes, but i think table creation is a low frequency operation, and adding this can help a lot when debugging some wierd problem.

Reply:
I think all the create/drop meta log should be printed. create/drop is low frequency operation.

Reply:
You guys can think a better way to print this log. You can create one line of log for creating table not each for one tablet.
Say if users want to create one table with 2000 partitions and each partition has 10 buckets. this will generate 20000 lines of log.
I don't think this is a good way to achieve your goal. So please do this in a more elegant way.

Reply:
OK, I will change it.